### DISCUSSION TITLE
Display Video NFTs in mobile & desktop wallet

### DISCUSSION THREAD

**Discussion by VDAODAO** [alephium/alephium-frontend#481] [Feature requests]
As of now, video NFTs are not visible in the NFT tab of the wallet. 
<img width="319" alt="Capture d’écran 2024-03-28 à 12 20 42" src="https://github.com/alephium/alephium-frontend/assets/42373571/39c20f58-8c2d-4e3f-9666-b48ea6a1033a">

<img width="300" src="https://github.com/alephium/alephium-frontend/assets/42373571/36866875-6a07-489e-b7ca-f9d1c467a5c3" />

<img width="300" src="https://github.com/alephium/alephium-frontend/assets/42373571/aa18a809-8dc6-40c7-8f14-3a1baae9d0de" />

**Comment by nop33** [comment]
This feature is already supported on desktop wallet and is implemented in the mobile wallet in this PR:
- https://github.com/alephium/alephium-frontend/pull/1223
Closing discussion.
